src/main/java/org/cryptomator/linux/tray/ActionItemCallback.java

```diff
@@ -0,0 +1,16 @@
+package org.cryptomator.linux.tray;
+
+import org.cryptomator.integrations.tray.ActionItem;
+import org.purejava.appindicator.GCallback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+record ActionItemCallback (ActionItem actionItem) implements GCallback {
+	private static final Logger LOG = LoggerFactory.getLogger(ActionItemCallback.class);
+
+	@Override
+	public void apply() {
+		LOG.debug("Hit tray menu action '{}'", actionItem.title());
```

Comment:
I think `trace` is sufficient for these kinds of events

Reply:
Done.